src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -133,10 +148,39 @@ private MongoConnection createConnection(ConnectionString cs, Properties info)
             throws SQLException {
         // attempt to get DIALECT property, and default to "mysql" if none is present
         String dialect = info.getProperty(DIALECT, MYSQL_DIALECT);
+        // Default log level is OFF
+        String logLevelVal = info.getProperty(LOG_LEVEL, Level.OFF.getName());
```

Comment:
In the rest of the code base we use `toLowerCase`. It's a small thing, but I'd rather be consistent (I don't care which one we choose!).

Reply:
Unfortunately, we can not use toLowerCase because log levels in java.utils.logging.Level are upper case. Using toLowerCase will make the parse method fail. I think here because the toUpperCase is applied to the properties values, it kind of makes sense and does not require changing what we do elsewhere.